data/data_loader.py

```diff
@@ -0,0 +1,43 @@
+
+from utils.config import top_config
+from torch.utils.data import Dataset, DataLoader
+import torch
+
+import os
+import sys
+
+'''
+Decouple the reference path.
+'''
+cwd = os.getcwd()  # current working directory
+cfp = os.path.dirname(os.path.abspath(__file__))  # current file path
+os.chdir(cfp)
+sys.path.append(os.path.abspath(".."))
+os.chdir(cwd)
+
+# TODO(diwei): modify
+class TopDataset(Dataset):
+    def __init__(self):
+        self.size = 10
+
+        self.xs = torch.rand(top_config["model_params"]["src_length"],
+                             self.size, top_config["model_params"]["d_model"])
+        self.us = torch.rand(top_config["model_params"]["enc_length"],
```

Comment:
"enc_length" appears to be a typo? it should refer to the sequence length of the decoder.

Reply:
`enc_length` means the length of the output from the encoder. It's a little ambiguous. Any advice?

Reply:
```suggestion
        self.us = torch.rand(top_config["model_params"]["tgt_length"],
``` 
I think it should refer to the sequence length to the decoder layer rather than the output length of encoder